benefit/data/sheets/AuthorisationCaseEvent/AuthorisationCaseEvent-nonprod.json

```diff
@@ -22,5 +22,7 @@
   {"LiveFrom": "22/10/2019", "CaseTypeID": "Benefit", "CaseEventID": "actionPostponementRequest", "UserRole": "caseworker-sscs", "CRUD": "R"},
   {"LiveFrom": "22/10/2019", "CaseTypeID": "Benefit", "CaseEventID": "actionPostponementRequestWelsh", "UserRole": "caseworker-sscs-systemupdate", "CRUD": "CRUD"},
   {"LiveFrom": "22/10/2019", "CaseTypeID": "Benefit", "CaseEventID": "actionPostponementRequestWelsh", "UserRole": "caseworker-sscs-superuser", "CRUD": "R"},
-  {"LiveFrom": "22/10/2019", "CaseTypeID": "Benefit", "CaseEventID": "actionPostponementRequestWelsh", "UserRole": "caseworker-sscs", "CRUD": "R"}
+  {"LiveFrom": "22/10/2019", "CaseTypeID": "Benefit", "CaseEventID": "actionPostponementRequestWelsh", "UserRole": "caseworker-sscs", "CRUD": "R"},
+  {"LiveFrom": "01/11/2021", "CaseTypeID": "Benefit", "CaseEventID": "confirmPanelComposition", "UserRole": "caseworker-sscs-judge", "CRUD": "CRUD"},
```

Comment:
think there should be a R option for user role caseworker-sscs

Reply:
as per AC only superuser and judge can access the event

Reply:
But if you don't add Read permission, it won't show in history of the case which creates confusion as they can't see everything that has happened on the case. We normally add Read permission as a minimum for all users for all events (e.g.  caseworker-sscs) unless there is a good reason not to

Reply:
Ok, added the R permission